octobot_commons/config_manager.py

```diff
@@ -110,19 +126,23 @@ def is_in_dev_mode(config):
     return CONFIG_DEBUG_OPTION in config and config[CONFIG_DEBUG_OPTION]
 
 
-# @staticmethod
-# def update_evaluator_config(to_update_data, current_config, deactivate_others):
-#     _update_activation_config(to_update_data, current_config,
-#                                             CONFIG_EVALUATOR_FILE_PATH, CONFIG_EVALUATOR_FILE,
-#                                             deactivate_others)
-#
-# @staticmethod
-# def update_trading_config(to_update_data, current_config):
-#     _update_activation_config(to_update_data, current_config,
-#                                             CONFIG_TRADING_FILE_PATH, CONFIG_TRADING_FILE,
-#                                             False)
+def update_evaluator_config(to_update_data, current_config, deactivate_others):
+    _update_activation_config(to_update_data,
+                              current_config,
+                              CONFIG_EVALUATOR_FILE_PATH,
+                              CONFIG_EVALUATOR_FILE,
+                              deactivate_others)
+
+
+def update_trading_config(to_update_data, current_config):
```

Comment:
Can we think about the way of having this code in OctoBot-Trading module as a module-config-manager ?

Reply:
https://github.com/Drakkar-Software/OctoBot-Trading/issues/35